content/en/real_user_monitoring/session_replay/mobile/_index.md

```diff
@@ -18,26 +18,97 @@ Mobile Session Replay expands visibility into your mobile applications by visual
+
+The first wireframe could dictate the viewport size, enabling player size adjustment for given device + rotation (landscape / portrait).
+
+| Iteration | 4 | 5-9 | 10-12 |
+|-----------|---|---|---|
+| Viewport | {{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-iteration-4.png" alt="An example of a 'geometry', 'image, and 'text' wireframe." style="width:100%;">}} | {{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-iteration-5-9-1.png" alt="An example of a 'geometry' and 'image' wireframe." style="width:100%;">}} | {{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-iteration-10-12.png" alt="An example of a 'geometry' and 'image' wireframe." style="width:100%;">}} |
+| Description | Draw "image" and "text" wireframes into viewport.| Draw "image" and "text" wireframes into viewport. | Draw "image" and two "geometry" wireframes into viewport.  |
+
+Because wireframes are sorted in back-to-front order, it overdraws existing portion of the frame. This is very much desirable. For semi-transparent elements, it can also do the job of proper blending "closer" elements with the "farer" ones.
+
+| Iteration | 13-19 | Final result |
+|-----------|-------|--------------|
+| Viewport | {{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-iteration-13-19.png" alt="An example of a 'geometry' and 'image' wireframe." style="width:50%;">}} | {{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-iteration-final-result.png" alt="An example of a 'geometry' and 'image' wireframe." style="width:50%;">}} |
+
+### Full and incremental snapshots
+
+The sequence of all visible wireframes can be considered a full snapshot record if it is referring to the web format. Sending it per frame in the replay is not optimal, which is why the recorder takes incremental snapshots.
+
+To send incremental changes, the mobile SDK tags each wireframe with a unique identifier. Native views are objects, sot hey can be identified by reference.
+
+{{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-wireframe-uuid.png" alt="An example of how each wireframe is given a `uuid` (from 1 to 19)." style="width:40%;">}}
+
+Except `uuid`, a wireframe can define 3 classes of attributes:
+
+- **Geometry**: Its `x`, `y`, `width`, and `height` using absolute screen coordinates.
+- **Appearance**: Information on its background color, font size, font color, and other styles.
+- **Content**: Depending on wireframe type, it can be a text from the label, reference to an image (such as an image's `uuid` or remote URL) or any other information essential for its rendering.
+
```

Comment:
```suggestion
```

I would remove the screenshot as it's redundant, and the piece about our data model as its too detailed and doesn't really provide any value for the reader.